### DISCUSSION TITLE
Change Region not work during deploy

### DISCUSSION THREAD

**Discussion by justoverclockl** [vercel/community#5064] [Help] [answered]
### Description

Hi all, today i've changed my Function Region to Frankfurt. After that i've re-deployed my app, but the log says:

`Running build in Washington, D.C., USA (East) – iad1 (Hive)`

How can i change the region correctly? I'm not using vercel.json.

**Comment by mountainash** [comment, marked as answer]
Build region is not the same as Function region. Builds happen on deployment, functions run on request (after your project has been deployed).

**Reply by amyegan** [reply]
Right, build region is not the same as runtime region. It looks like the Function Region was probably already set correctly in Project Settings.

You can find more info about setting the region for serverless functions in the docs: [vercel.com/docs/functions/configuring-functions/region](https://vercel.com/docs/functions/configuring-functions/region)

**Comment by mountainash** [comment]
@justoverclockl you might want to look at the request to allow Build Region configuration, as this might be a feature that benefits you too: [#10888](https://github.com/vercel/vercel/discussions/10888)
